File: fatorial.py
# Calcula o fatorial de um número.
# Calculates the fatorial of a number.

def Fatorial(fat):
	if (fat == 1):
	    return 1
	return fat * Fatorial(fat - 1)

n = (int(input("Digite um número: ")))

print("Fatorial de {} é: {}".format(n,Fatorial(n)))

File: fatorial.rb
#
# autor Maurício Freire
# Calcula o fatorial de um número.
# Calculates the fatorial of a number.

def fatorial(n)
    if n == 1 
        return 1
    end
    return n * fatorial(n - 1)
end

n = gets.chomp.to_i
puts "Fatorial de #{n} = #{fatorial(n)}"
